### PR TITLE
Morse decode

### PR DESCRIPTION
**Pull request by principles31**
# Decode a Morse code message:
## In this PR the following were implemented with pair programming,

> Create a method to decode an entire word in Morse code, takes a string parameter, and return the string representation. - Every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").

> Create a method to decode the entire message in Morse code, takes a string parameter, and return the string representation. Every word will be separated by 3 spaces (e.g.

 -  decode("-- -.--   -. .- -- .")
- returns "MY NAME").

- Reuse methods whenever possible. Make sure that your code is DRY.

- Once your code is ready use it to decode the message from the old bottle:

      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...
- You can use IRB for that :)

### CODE REVIEW

File: morse_decode.rb [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+end
+
+def decode_word(word)
+  @entire_word = ''
+  @chars = word.split
+  @chars.each do |char|
+    @entire_word += decode_char(char)
+  end
+  print @entire_word.to_s
+end
```
Comment (codepantha) [inline morse_decode.rb:53]: - [x] You are almost there 👏 but currently you don't account for spaces in the words. When I try to decode a sentence, they are squished together. Kindly see below and try to fix it 👇 
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️ 

## Hi team 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights 🎯 
✔️ Descriptive PR
✔️ No linter errors
✔️ Correct GitFlow

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@codepantha Thanks for the feedback

**Review by principles31 — COMMENTED** [review]
- You are almost there clap but currently you don't account for spaces in the words. When I try to decode a sentence, they are squished together. Kindly see below and try to fix it point_down
-  The above changes required by the previous reviewer have been implemented to the best of my knowledge.

**Review by rotimiazeez — APPROVED** [review]
# Hi TEAM 👍🏽 , 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by principles31** [comment]
@rotimiazeez Thank you.

**Pull request merged**